src/config.ts

```diff
@@ -32,6 +36,9 @@ export class Config {
 
         config.logPath = this.param(args, 'logPath')
 
+        config.clientKeepAliveInterval = +this.param(args, 'clientKeepAliveInterval', '0')
```

Comment:
Please put the serverKeep alive with the other server vars, and then client below